graphql-cli/src/main/java/io/ballerina/graphql/generator/ballerina/ConfigTypesGenerator.java

```diff
@@ -0,0 +1,324 @@
+/*
+ *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2022, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
```

Reply:
Updated in https://github.com/ballerina-platform/graphql-tools/commit/89275182a49a4570dc81854b7b7ba535a608f613